Formula/redis@2.8.rb

```diff
@@ -0,0 +1,74 @@
+class Redis28 < Formula
+  desc "Persistent key-value database, with built-in net interface"
+  homepage "http://redis.io/"
+  url "http://download.redis.io/releases/redis-2.8.24.tar.gz"
+  sha256 "f4c8e346e46a84daa3050db4f11123c66cee0346f6af7f4ff739072889e4e3fb"
+  head "https://github.com/antirez/redis.git", :branch => "2.8"
+
+  bottle do
+    cellar :any
+    sha256 "a116b30423226cd4445de190acd97c955962d7ae589ccac0aaf9b9585fb5dfca" => :yosemite
+    sha256 "a0b72d84d805f8fec1bc1307479ba550e620fca0198746704c23b44833e3e0bd" => :mavericks
+    sha256 "82d23051ab34705cf98a00e8416ec4ea394f5f1753b82519e8f0cfc36f87f46d" => :mountain_lion
+  end
+
+  keg_only :versioned_formula
+
+  def install
+    # Architecture isn't detected correctly on 32bit Snow Leopard without help
+    ENV["OBJARCH"] = MacOS.prefer_64_bit? ? "-arch x86_64" : "-arch i386"
+
+    # Head and stable have different code layouts
+    src = (buildpath/"src/Makefile").exist? ? buildpath/"src" : buildpath
+    system "make", "-C", src, "CC=#{ENV.cc}"
+
+    %w[benchmark cli server check-dump check-aof sentinel].each { |p| bin.install src/"redis-#{p}" => "redis28-#{p}" }
```

Comment:
I guess `redis@2.8 instead of `redis28`? And will it affect existing installations?